custom/icds/tasks/sms.py

```diff
@@ -51,3 +55,27 @@ def send_monthly_sms_report():
         send_html_email_async.delay(subject, recipients, message,
                                     email_from=settings.DEFAULT_FROM_EMAIL)
         raise e
+
+
+@receiver(post_save, sender=DailyOutboundSMSLimitReached)
+def send_sms_limit_exceeded_alert(sender, instance, **kwargs):
+    if settings.SERVER_ENVIRONMENT == 'icds':
+        sms_daily_limit = 10000
+        domain_obj = Domain.get_by_name(instance.domain)
+        if domain_obj:
+            sms_daily_limit = domain_obj.get_daily_outbound_sms_limit()
+
+        subject = "SMS Daily Limit exceeded"
+        recipients = [
+            'ndube@dimagi.com',
+            'dsivaramakrishnan@dimagi.com',
+            'mshashtri@dimagi.com',
+            'asharma@dimagi.com'
+        ]
+        message = _("""
+        Hi,
+        This is to inform you that the Daily SMS limit for domain {domain} has exceeded for {date}.
+        The current limit set is {sms_limit}
+        """).format(domain=instance.domain, date=instance.date, sms_limit=sms_daily_limit)
```

Comment:
nit: though it won't happen for ICDS, but if you want to handle cases when domain is empty you could do
```
if domain_obj:
    message = _("""
        Hi,
        This is to inform you that the Daily SMS limit for domain {domain} on environment {env} has exceeded for {date}.
        The limit was {sms_limit} per day.
    """).format(domain=instance.domain, env=settings.SERVER_ENVIRONMENT, date=instance.date, sms_limit=sms_daily_limit)
else:
    message = _("""
        Hi,
        This is to inform you that the outbound Daily SMS limit has exceeded for environment {env} for {date}.
        The limit was {sms_limit} per day.
    """).format(env=settings. SERVER_ENVIRONMENT , date=instance.date, sms_limit=sms_daily_limit)
```

Reply:
Fair point.
Thanks for noticing that.